README.md

```diff
@@ -37,6 +37,12 @@ features like flow control, data structures, and functions, with high-level
 constructs like objects, protocols, closures, and generics. Swift embraces
 modules, eliminating the need for headers and the code duplication they entail.
 
+Announced in 2014, the Swift programming language has quickly become one of the fastest growing languages in history.
+Swift makes it easy to write software that is incredibly fast and safe by design
+Swift is a powerful and intuitive programming language for iOS, iPadOS, macOS, tvOS, and watchOS.
```

Comment:
I think this line makes the description _less_ precise, and furthers this misconception that it's a Apple/Darwin-only language.

The rest of the additions are already covered on the front page of swift.org.

Reply:
@WowbaggersLiquidLunch thanks for reaching as you mentioned your point I support your point

 but I don't think so it's less precise form anypoint of view, regarding swift.org well if it's in README it won't affect them either

Reply:
> I think this line makes the description _less_ precise, and furthers this misconception that it's a Apple/Darwin-only language.
> 
> 
> 
> The rest of the additions are already covered on the front page of swift.org.

@WowbaggersLiquidLunch I concur with this, Swift can be used for a lot more than developing apps for Apple products.

Reply:
I agree this isn't very good wording, is a great _general purpose_ language and there is no need to highlight apple platforms at the exclusion of others like this.